src/objecttypes/tests/test_objecttype_admin.py

```diff
@@ -194,18 +194,6 @@ def test_update_draft(self):
         object_version.refresh_from_db()
         self.assertEqual(object_version.json_schema, JSON_SCHEMA)
 
-    def test_update_published_no_buttons(self):
```

Comment:
Might be a bit excessive, but I guess you could add a test to explicitly test if editing the metadata is possible (for both drafts and published types)

Reply:
the tests are added